packages/playground/src/domains/agent-builder/components/agent-edit/agent-profile/__tests__/tools-loading.test.tsx

```
@@ -0,0 +1,87 @@
+// @vitest-environment jsdom
+import type { ListToolProvidersResponse } from '@mastra/client-js';
+import { MastraReactProvider } from '@mastra/react';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { cleanup, render, screen, waitFor } from '@testing-library/react';
+import { http, HttpResponse } from 'msw';
+import type { ReactNode } from 'react';
+import { FormProvider, useForm } from 'react-hook-form';
+import { afterEach, describe, expect, it } from 'vitest';
+
+import { server } from '@/test/msw-server';
```

ESLint

[error] `@/test/msw-server` import should occur after import of `../tools`

(import/order)